site/LAYOUT.md

```diff
@@ -84,31 +84,41 @@ It allows programmatically determining which versions of Jenkins get served whic
 
 ## Regular tiered update sites (LTS and weekly)
 
-Directories with names containing Jenkins LTS baselines like `2.204` are version specific update sites.
+The Jenkins update center provides multiple update sites.
+Each of them only offer plugins compatible with the respective baseline, as well as the newest weekly or LTS release, depending on the variant (LTS or weekly).
+This increases the usefulness of the update sites to users of slightly older releases of Jenkins, as they won't regularly be offered incompatible releases of plugins.
 
-* `2.204/` contains the tiered update site for Jenkins 2.204 and _similar_ weekly releases not matching an LTS baseline.
-* `stable-2.204/` contains the tiered update site for Jenkins 2.204.x.
+---
+**NOTE**:  The update center used to support formats like `2.204/` and `stable-2.204/`.
+This is no longer a case for the recent versions.
 
-Each of these update sites only offer plugins compatible with the respective baseline, as well as the newest weekly or LTS release, depending on the variant (LTS or weekly).
-This increases the usefulness of the update sites to users of slightly older releases of Jenkins, as they won't regularly be offered incompatible releases of plugins.
+---
 
 The Jenkins project publishes a limited, fixed number of tiered update sites to encourage users to regularly update Jenkins. 
+Directories with names containing Jenkins LTS baselines like `2.319` are version specific update sites:
+
+* `dynamic-stable-2.319.1/` contains the tiered update site for Jenkins 2.319.x.
+
+By default, the update center generates versions only for the first `.1` release of the LTS baseline.
+We expect binary and API compatibility within the LTS baseline,
+so it is extremely unlikety there would be a need for a specific update site for a `.2` release.
```

Comment:
```suggestion
We expect binary and API compatibility within the LTS baseline, so it is extremely unlikely there would be a need for a specific update site for a `.2` release.
```

Reply:
Why? It is better for editing to keep short lines

Reply:
Why keeping short lines?
Especially since browsers control the line flow.

Note: there is also a typo fix on "unlikely" in this suggestion.

Reply:
> Why keeping short lines?
Especially since browsers control the line flow.

Editing diffs becomes much easier, and there are less merge conflicts when it comes to collaborative editing

Reply:
Not convinced it's better, but I won't block your PR for this.
Feel free to mark this conversation as resolved, I'll make another suggestion for the typo.